curriculum/js3/1.js

```diff
@@ -4,11 +4,21 @@
  * @returns {number} max
  */
 
+const MaxValue=(arr,i=0, max=arr[0])=>{
```

Comment:
The idea of this question is to iterate through an object with its keys.

You should be iterating over `Object.keys()` instead.